bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZprimeVBF_tau_tau/proc_card_mg5.template

```diff
@@ -1,5 +1,5 @@
 import model_v4 $modelfile
 define p = g u c d s u~ c~ d~ s~
 define j = g u c d s u~ c~ d~ s~
-generate p p > zp j j  ,  zp > ta- ta+ 
+generate p p > zp j j  ,  zp > ta- ta+ QCD=0 @0
```

Comment:
QCD=0 should've been in the middle, "generate p p > zp j j QCD=0, zp > ta+ ta-" for VBF processes
This model card you use luckily forbids zprime coupling with light quarks so you don't have unwanted gluon mediated processes, you only get VBF. But for future reference, can you fix this like above?

Reply:
done

Reply:
@aman0404 I think we already have this fixed in your gridpacks but I believe it should be
 generate p p > zp j j  QCD=0,  zp > ta- ta+ @0
NOT  generate p p > zp j j  ,  zp > ta- ta+ QCD=0 @0
right??